libafl_qemu/src/modules/edges.rs

```diff
@@ -357,6 +357,8 @@ pub struct EdgeCoverageModuleBuilder<AF, PF, V> {
 pub struct EdgeCoverageModule<AF, PF, V> {
     variant: V,
     address_filter: AF,
+    // we only use it in system mode at the moment.
+    #[cfg_attr(not(emulation_mode = "systemmode"), allow(dead_code))]
```

Comment:
more like it doesn't make sense to have this member in usermode.
i don't think we will ever need a page filter for normal processes.

Reply:
yes.
didn't do it yet because i think we'll have to do code duplication because of this